policy/predicate/file.go

```diff
@@ -33,11 +55,9 @@ type ChangedFiles struct {
 var _ Predicate = &ChangedFiles{}
 
 func (pred *ChangedFiles) Evaluate(ctx context.Context, prctx pull.Context) (*common.PredicateResult, error) {
-	var paths, ignorePaths []string
+	var ignorePaths []string
```

Comment:
Thanks for adding the `getPathStrings` function - I think we can also use it to compute `ignorePaths` which is basically the same loop with a different input.